### DISCUSSION TITLE
Cannot find namespace 'DataTables'.

### DISCUSSION THREAD

**Discussion by aungpt** [l-lin/angular-datatables#1784] [General]
I used this version since 3 months age 
but yesterday I deleted node_modules and .angular folder and then I run npm install.
after that ng build facing issue any suggestion for it.    

```
` "angular-datatables": "^13.1.0",
    "angular2-moment": "^1.9.0",
    "bootstrap": "^3.4.1",
    "core-js": "^3.6.4",
    "datatables-bootstrap": "0.0.1",
    "datatables-bootstrap3-plugin": "^0.6.0",
    "datatables.net": "^1.10.20",
    "datatables.net-bs": "^1.10.20",
    "datatables.net-dt": "^1.10.20",`
```
    
![image](https://github.com/l-lin/angular-datatables/assets/52529847/2c986903-b3da-4d7e-b275-a097425302d8)

**Comment by Thomas14R** [comment]
Hi,

This is due to the dev dependency [@types/datatables.net](https://www.npmjs.com/package/@types/datatables.net) being deprecated a few days ago now since datatables includes its own type definitions since v1.11.
I'd recommend reading through this discussion here: #1777 and the PR: #1783

I believe you can download and build the dt-2 branch of this repo locally and include it in your node_modules/angular-datatables.
I'm unsure if specifying the version of @types/datatables.net in your packages.json to 
`"@types/datatables.net": "1.10.28",` would work but it might be worth a try.

Hopefully this helped you!

**Reply by aungpt** [reply]
thanks bro
now working.

**Reply by max-l-weaver** [reply]
Had same problem and installed `"@types/datatables.net": "1.10.28",` and it now works.

Thanks for the suggestion
